MesnardBarba-CiSE.md

```diff
@@ -180,48 +181,48 @@ Not so fast!
 We rely on external libraries to solve sparse linear systems of equations: _Cusp_ for GPU devices and PETSc for distributed CPU systems. 
 It turns out, the iterative solvers may have differences that affect the final solution.
 
-When repeating our previous simulations of the aerodynamics of a snake cross-section with PetIBM, the solutions do not always match those with cuIBM. 
+When repeating our previous simulations of the aerodynamics of a snake cross-section with PetIBM, the solutions do not always match those computer with cuIBM. 
 At a Reynolds number of 1000, both the time-averaged lift and drag coefficients match. 
 But at Reynolds equal to 2000, average lift and drag match up to 30 degrees angle-of-attack, but not at 35 degrees. 
 That means that we don't see lift enhancement (Figure 8) and the main finding of our previous study is not fully replicated. 
 Looking at the time evolution of the force coefficients for the simulation with PetIBM at Re=2000 and 35 degrees angle-of-attack, we see a marked drop in lift after 35 time units (top graph in Figure 9). 
 What is different in the two codes? 
 Apart from using different linear algebra libraries, they run on different hardware. 
 Leaving hardware aside for now, let's focus on the iterative solvers. 
-Both _Cusp_ and PETSc use the same convergence criterion. 
+Both _Cusp_ and PETSc use the same convergence criterion.  (Huh? Maybe this should be posed as a question?)
```

Comment:
Not a question. They do, in fact, define their convergence criterion in the same manner. Confirmed by @piyueh.